src/main/java/io/jenkins/plugins/monitoring/IMonitorPortlet.java

```diff
@@ -0,0 +1,62 @@
+package io.jenkins.plugins.monitoring;
+
+import java.util.Optional;
+
+/**
+ * This interface defines the portlet to be shown in the monitoring dashboard.
+ *
+ * @author Simon Symnhoven
+ */
+interface IMonitorPortlet {
```

Comment:
I would not recommend to use `I` to represent an interface. Where do we need the interface? WOuldn't it be simpler to use the abstract class as the interface?

Reply:
Yes that makes it easier!